apps/guest/src/home/components/modal-qualify/modal-qualify.component.html

```diff
@@ -15,7 +16,9 @@
     (rate)="selectStart($event)"
   ></contler-start>
 
-  <p class="cnt-qualify__info" *ngIf="value !== undefined">{{ textRate[value - 1] }} {{ value }}/5</p>
+  <p class="cnt-qualify__info" *ngIf="value !== undefined">
+    {{ 'modalQualifyComponent.' + textRate[value - 1] | translate }} {{ value }}/5
```

Comment:
```suggestion
    {{ textRate[value - 1] | translate }} {{ value }}/5
```
For example, it is easier to write